homeassistant/components/lutron_caseta.py

```diff
@@ -41,9 +41,7 @@ def setup(hass, base_config):
 
     config = base_config.get(DOMAIN)
     hass.data[LUTRON_CASETA_SMARTBRIDGE] = Smartbridge(
-        hostname=config[CONF_HOST],
-        username=config[CONF_USERNAME],
-        password=config[CONF_PASSWORD]
```

Comment:
We should not allow any config options that are not used. Please remove them and let's mark this a breaking change.